GOVERNANCE.md

```diff
@@ -76,9 +76,14 @@ Subsequent terms will be for a period of 1 year ending the following March 31st.
 ## Term for CPC Board representatives
 
 As per the CPC charter, the term for CPC Board representatives is 1 year. Due to the bootstrap
-process, the term for the first year will be adjusted such that the initial term for all CPC
-Board representatives will end June 30th, 2020. Subsequent terms will be for a period
-of 1 year ending the following June 30th. 
+process and the decision to stagger CPC representative terms by 6 months, the initial term for the
+[Primary CPC Director][] will end February 28, 2021 and the initial term for the
+[Secondary CPC Director][] will end August 31, 2020. Subsequent terms will be for a period
+of 1 year ending the last day of February for the Primary CPC Director and August 31 for the
+Secondary CPC Director.
```

Comment:
Did we choose the June 30th date previously for a particular reason? I just want to ask to make sure we aren't overlooking something that we were planning around or incoordination with.

Reply:
I discussed with @brianwarner and there doesn't seem to be any issue with these dates or any reason why we had chosen the previous dates. So .... 👍